runtime/client_http_request.go

```diff
@@ -61,6 +61,7 @@ func NewClientHTTPRequest(
 		MethodName:     methodName,
 		client:         client,
 		Logger:         client.loggers[methodName],
+		ContextLogger:  NewContextLogger(client.loggers[methodName]),
```

Comment:
Now that we have a proper context logger, we will no longer need to create a child logger per method as in https://github.com/uber/zanzibar/blob/master/runtime/http_client.go#L62-L68, we only need to include the method name in the context log fields. That could be in different PR tho (please create an issue to remind us if you decide to have that done separately).

Reply:
Yes, it'll be done in a separate diff.